manim/animation/creation.py

```diff
@@ -133,6 +233,8 @@ def interpolate_submobject(self, submob, start, outline, alpha):
 
 
 class Write(DrawBorderThenFill):
+    """Simulate hand-writing a :class:`~.Text`."""
```

Comment:
Maybe we can even add an example for all off these?

Reply:
And I think this can be used by more Mobjects then only Text.

Reply:
Yes, in the future I would add examples to everything. However, the purpose of this PR is just to have *something* here.

Reply:
okay, good point!
> And I think this can be used by more Mobjects then only Text. See for example here:
https://docs.manim.community/en/latest/examples.html#threedfunctionplot
self.play(Write(plane))

But then I think It would be nice to add some more text to indicate that Write is not for Text only

Reply:
Oh woops sorry I missed that.